typescript/sdk/src/token/EvmERC20WarpRouteReader.ts

```diff
@@ -992,13 +992,14 @@ export class EvmERC20WarpRouteReader extends EvmRouterReader {
     try {
       return await contractWithVersion.PACKAGE_VERSION();
     } catch (err: any) {
-      if (err.cause.code && err.cause.code === 'CALL_EXCEPTION') {
+      if (err.cause?.code && err.cause?.code === 'CALL_EXCEPTION') {
         // PACKAGE_VERSION was introduced in @hyperlane-xyz/core@5.4.0
         // See https://github.com/hyperlane-xyz/hyperlane-monorepo/releases/tag/%40hyperlane-xyz%2Fcore%405.4.0
         // The real version of a contract without this function is below 5.4.0
         return '5.3.9';
       } else {
-        throw err;
+        this.logger.error(`Error when fetching package version ${err}`);
+        return '0.0.0';
```

Comment:
Remind me, why are we returning version 0 instead of the version before the `PACKAGE_VERSION` method was introduced?

Reply:
we can also return '5.3.9'. i was just distinguishing between an expected error that returns '5.3.9' versus a non-expected one that returns '0.0.0'